src/main/java/by/glinkiy/lesson2/task8/Rubles.java

```diff
@@ -0,0 +1,33 @@
+package by.glinkiy.lesson2.task8;
+
+import java.util.Random;
+
+/**
+ * Имеется целове число (задать с помощью Random rand = new Random(); int x =
+ * rand.nextInt() ). Это число – количесво денег в рублях. Вывести это число, добавив к
+ * нему слово «рублей» в  правильном падеже.
+ */
+public class Rubles {
+    public static void main(String[] args) {
+        Random rand = new Random();
+        int x = rand.nextInt();
```

Comment:
of course I expect positive number of money
-1962871473 Digit don't must be negative - looks not so good(
as you should do task without java libs (such as Math) so optional solution will be check if < then then 0 and multiply by -1